CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributor Onboarding
+
+🚧 This document is still a WIP 🚧 
+
+## General
+
+ - The goal of this guide is to help you start contributing to Durable Functions
+
+## Pre-reqs
+
+ - OS
+    - Windows 10 (suggested)
+    - Mac OS X/Linux
```

Comment:
I know it's possible to run the functions CLI on macOS or Linux, but is it possible to actually build WebJobs.Extensions.DurableTask.csproj on macOS or Linux? I'm actually not sure since I'm not familiar with how well the VS tooling works on these platforms.

Reply:
Good point. Let's remove it for now. Later we can extend after we test the experience on Mac and Linux.